blueflood-http/src/test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedIngestionTests.java

```diff
@@ -18,33 +18,36 @@
 
 import com.google.gson.internal.LazilyParsedNumber;
 import com.netflix.astyanax.serializers.AbstractSerializer;
+import com.rackspacecloud.blueflood.inputs.formats.JSONMetricsContainerTest;
 import com.rackspacecloud.blueflood.inputs.handlers.wrappers.AggregatedPayload;
+import com.rackspacecloud.blueflood.inputs.handlers.wrappers.TestGsonParsing;
 import com.rackspacecloud.blueflood.io.serializers.NumericSerializer;
+import com.rackspacecloud.blueflood.service.Configuration;
+import com.rackspacecloud.blueflood.service.CoreConfig;
 import com.rackspacecloud.blueflood.types.PreaggregatedMetric;
-import junit.framework.Assert;
 import org.junit.Before;
 import org.junit.Test;
 
-import java.io.BufferedReader;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.io.InputStreamReader;
+import java.io.*;
 import java.util.Collection;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import static junit.framework.Assert.*;
+
 
 public class HttpAggregatedIngestionTests {
-    
+
+    public static final long TIME_DIFF = 2000;
+
     private AggregatedPayload payload;
-    
+
+    private final String  prefix = "pref.";
+
     @Before
     public void buildPayload() throws IOException {
-        StringBuilder sb = new StringBuilder();
-        BufferedReader reader = new BufferedReader(new InputStreamReader(new FileInputStream("src/test/resources/sample_payload.json")));
-        String curLine = reader.readLine();
-        while (curLine != null) {
-            sb = sb.append(curLine);
-            curLine = reader.readLine();
-        }
-        String json = sb.toString();
+
+        String json = TestGsonParsing.getJsonFromFile( new InputStreamReader(new FileInputStream("src/test/resources/sample_payload.json")), prefix );
```

Comment:
Yeah, this will break when run from IntelliJ. I'm surprised I missed this when I changed the other ones to load from Classpath. Can you please copy from the other code that loads the Json file from Classpath?